Documentation/dev-tools/kfence.rst

```diff
@@ -281,9 +287,11 @@ found in the userspace `Electric Fence Malloc Debugger
 In the kernel, several tools exist to debug memory access errors, and in
 particular KASAN can detect all bug classes that KFENCE can detect. While KASAN
 is more precise, relying on compiler instrumentation, this comes at a
-performance cost. We want to highlight that KASAN and KFENCE are complementary,
-with different target environments. For instance, KASAN is the better
-debugging-aid, where a simple reproducer exists: due to the lower chance to
-detect the error, it would require more effort using KFENCE to debug.
-Deployments at scale, however, would benefit from using KFENCE to discover bugs
-due to code paths not exercised by test cases or fuzzers.
+performance cost.
+
+We want to highlight that KASAN and KFENCE are complementary, with different
```

Comment:
Not sure we need to write in first person here, it's only a piece of documentation, not a patch description or an article.
How about: "It is worth highlighting that..."?

Reply:
Done.